json_schemas/services-digital-outcomes-and-specialists-user-research-studios.json

```diff
@@ -1,7 +1,134 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "additionalProperties": true,
+  "additionalProperties": false,
```

Comment:
Not a big deal, but the order of the keys is weird. We could make it more logical (ie. title and description near the top) by generating the data from a `collections.OrderedDict`.